### PR TITLE
Fabo/handle deposit timeline

### PR DESCRIPTION
**Pull request by faboweb**
Closes #ISSUE

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI
image

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5f64f7e0145ec0144559b907--lunieio.netlify.app

File: api/lib/source/cosmosV0-source.js [hunk 1]
```diff
@@ -310,25 +310,32 @@ class CosmosV0API extends RESTDataSource {
...
               title: `Voting Period Ended`,
-              time: proposal.voting_end_time
+              time:
+                proposal.voting_end_time !== `0001-01-01T00:00:00Z`
+                  ? proposal.voting_end_time
+                  : undefined
             }
           : undefined
-      ]
+      ].filter((x) => !!x)
```
Comment (jbibla) [inline api/lib/source/cosmosV0-source.js:338]: 😨

**Review by jbibla — APPROVED** [review]

File: api/lib/source/cosmosV0-source.js (hunk 1 above)
Comment (faboweb) [reply api/lib/source/cosmosV0-source.js:338]: filter out undefineds

Comment (faboweb) [reply api/lib/source/cosmosV0-source.js:338]: filter out undefineds. actually not needed

File: api/lib/source/cosmosV0-source.js [hunk 2]
```diff
@@ -310,25 +310,32 @@ class CosmosV0API extends RESTDataSource {
               title: `Deposit Period Ended`,
               // the deposit period can end before the time as the limit is reached already
               time:
+                proposal.voting_start_time !== `0001-01-01T00:00:00Z` &&
```
Comment (Bitcoinera) [inline api/lib/source/cosmosV0-source.js:313]: what is that weird date?

Comment (faboweb) [reply api/lib/source/cosmosV0-source.js:313]: so time = undefind

Comment (Bitcoinera) [reply api/lib/source/cosmosV0-source.js:313]: aah. Wow :eyes: Javascript can be very crazy

**Pull request merged**